web/shared/src/poll_data.ts

```diff
@@ -1,3 +1,73 @@
+import z from "zod";
+
+export type PollDataConfig = {
+    message_sender_id: number;
+    current_user_id: number;
+    is_my_poll: boolean;
+    question: string;
+    options: string[];
+    comma_separated_names: (user_ids: number[]) => string;
+    report_error_function: (msg: string, more_info?: unknown) => void;
+};
+
+export type PollOptionData = {
+    option: string;
+    names: string;
+    count: number;
+    key: string;
+    current_user_vote: boolean;
+};
+
+type PollOption = {
+    option: string;
+    user_id: string | number;
```

Comment:
Is it possible to get a `string` for `user_id`?

Reply:
Since it comes from `event.sender_id`, could you confirm that it will always be a `number`?

Reply:
It's coming from passing "canned" manually.